src/app/tournament/bracket/bracket/bracket.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { DomSanitizer} from '@angular/platform-browser';
+
+@Component({
+  selector: 'tyl-bracket',
+  templateUrl: './bracket.component.html',
+  styleUrls: ['./bracket.component.scss']
+})
+export class BracketComponent implements OnInit {
+
+  constructor(private sanitizer: DomSanitizer) { }
+
+  @Input() key: string;
+
+  ngOnInit(): void {
+  }
+
+  get url() {
+    return this.sanitizer.bypassSecurityTrustResourceUrl(`https://challonge.com/${this.key}/module?show_tournament_name=1`);
```

Comment:
Not a big deal, but this should be exposed as a property.  While in reality this wont get checked more than once, should the dom structure change where rendering changes happen higher in the dom, it will cause this function to be called more than once.  It's a better practice to just expose this as a property so change detection can cache this value.

Reply:
I moved this into a property, I think it actually was causing a problem the other way